Include/GWCA/Packets/StoC.h

```diff
@@ -1036,7 +1036,65 @@ namespace GW {
                 uint32_t title_id;
                 uint32_t new_value;
             };
-            constexpr uint32_t Packet<UpdateTitle>::STATIC_HEADER = GAME_SMSG_TITLE_UDPATE;
+            constexpr uint32_t Packet<UpdateTitle>::STATIC_HEADER = GAME_SMSG_TITLE_UPDATE;
+
+            struct TitleInfo : Packet<TitleInfo> {
+                uint32_t title_id;
+                uint32_t unk_flags;
+                uint32_t value;
+                uint32_t h000c; // Maybe an id for current rank
+                uint32_t current_rank_minimum_value;
+                uint32_t h0014; // Maybe an id for next rank
+                uint32_t next_rank_minimum_value;
+                uint32_t num_ranks;
+                uint32_t h0020; // Maybe an id for e.g. base rank?  Matches h000c for wisdom on my r1 wisdom alt acc
+                wchar_t progress_display_enc_string[8];
+                wchar_t mouseover_hint_enc_string[8];
+            };
+            constexpr uint32_t Packet<TitleInfo>::STATIC_HEADER = GAME_SMSG_TITLE_TRACK_INFO;
+
+            namespace PlayerAttrId {
+                constexpr uint32_t EXPERIENCE = 0;
```

Comment:
wouldn't this better be an enum class?

Reply:
Probably.  Originally it was just a single constant for imperial cap then I missed this opportunity while refactoring to contribute.